docs/BreakingChanges/119 ADO.NET idle connection resiliency.md

```diff
@@ -0,0 +1,39 @@
+## 119: ADO.NET idle connection resiliency
```

Comment:
{
  "Id": "119",
  "Title": "ADO.NET idle connection resiliency",
  "Details": "Beginning in the .NET Framework 4.5.1, the .NET Framework will attempt to automatically reconnect broken SQL connections. Although this will typically make apps more reliable, there are edge cases in which an app needs to know that the connection was lost so that it can take some action upon reconnection.",
  "Suggestion": "If this feature is undesirable due to compatibility concerns, it can be disabled by setting the ConnectRetryCount property of a connection string (or SqlConnectionStringBuilder) to 0.",
  "Link": "http://blogs.msdn.com/b/dotnet/archive/2013/10/17/net-framework-4-5-1-rtm-gt-start-coding.aspx",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Data.IDbConnection.set_ConnectionString(System.String)",
    "M:System.Data.SqlClient.SqlConnection.set_ConnectionString(System.String)",
    "M:System.Configuration.ConnectionStringSettings.set_ConnectionString(System.String)",
    "M:System.Data.Common.DbConnection.set_ConnectionString(System.String)",
    "M:System.Data.Common.DbConnectionStringBuilder.set_ConnectionString(System.String)",
    "M:System.Data.SqlClient.SqlConnectionStringBuilder.#ctor",
    "M:System.Data.SqlClient.SqlConnectionStringBuilder.#ctor(System.String)",
    "M:System.Data.Common.DbConnectionStringBuilder.#ctor",
    "M:System.Data.Common.DbConnectionStringBuilder.#ctor(System.Boolean)"
  ],
  "Related": null,
  "VersionBroken": "4.5.1",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": "It would be very difficult to know if code depended on not auto-reconnecting and ADO.NET connection.\nSource analyzer status: Planned - No tracking work item yet\nSingle-diagnostic compilation action analyzer",
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Data.IDbConnection.set_ConnectionString(System.String)
- [x] M:System.Data.SqlClient.SqlConnection.set_ConnectionString(System.String)
- [x] M:System.Configuration.ConnectionStringSettings.set_ConnectionString(System.String)
- [x] M:System.Data.Common.DbConnection.set_ConnectionString(System.String)
- [x] M:System.Data.Common.DbConnectionStringBuilder.set_ConnectionString(System.String)
- [x] M:System.Data.SqlClient.SqlConnectionStringBuilder.#ctor
- [x] M:System.Data.SqlClient.SqlConnectionStringBuilder.#ctor(System.String)
- [x] M:System.Data.Common.DbConnectionStringBuilder.#ctor
- [x] M:System.Data.Common.DbConnectionStringBuilder.#ctor(System.Boolean)